data/wanderer/wanderers start.txt

```diff
@@ -1998,12 +2000,26 @@ mission "Wanderers Hai Assistance Fleet"
 	invisible
 	to offer
 		has "Wanderers Hai Assistance 1: done"
+	on offer
+		event "hai blockade"
 	npc kill
 		government "Hai"
 		personality heroic staying uninterested frugal
 		system "Wah Ki"
 		fleet "Large Hai" 6
 
+event "hai blockade"
+	government "Hai (Raided)"
+		remove raid
```

Comment:
Is the idea that they stop Raiding because they have to consolidate fleet efforts in Wanderer space and work out how to deal with the Arfectas?

Reply:
Its because the Hai set up a big blockade in Wah Ki, killing ships attempting to go trough
I think the Hai should remove it at some point, restoring the raids?
Uhai dont lack ships I think